packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -15,18 +27,58 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
   TreeTable,
   UITreeRow
 > {
-  // table: TreeTable
+  /** We keep a virtual column at the front that tracks the "group" that is expanded */
+  private virtualColumns: Column[];
+
+  constructor(
+    dh: DhType,
+    table: TreeTable,
+    formatter = new Formatter(dh),
+    inputTable: InputTable | null = null
+  ) {
+    super(dh, table, formatter, inputTable);
+    this.virtualColumns = [
+      {
+        name: 'Group',
```

Comment:
Yea, I'm tempted to put a special char in there so that it doesn't have the possibility of conflicting with another column name... or instituting an optional `displayName` property on Columns that we use for rendering. Because a name conflict could cause problems... 
We do have a ticket for having a display name for columns: https://github.com/deephaven/deephaven-plugins/issues/105
I'm tempted to just add support for that, but that's also not spec'ed out and looks like it'll be wired up using `ui.table`. So worried whatever I do support for displayName here will not work correctly with whatever we implement with `deephaven.ui`.

Reply:
I added support for `displayName`.